schedule/sles4sap/ensa/ensa_nodes.yaml

```diff
@@ -49,6 +49,7 @@ schedule:
   - sles4sap/ensa/netweaver_ensa2_cluster
   - sles4sap/ensa/netweaver_ensa2_cluster_connector
   - sles4sap/ensa/netweaver_ensa2_web_methods
+  - sles4sap/sap_suse_cluster_connector
```

Comment:
Sure, I will do the code changes in ticket "[TEAM-9633](https://jira.suse.com/browse/TEAM-9633)
Rename ensa modules "netweaver_ensa2_cluster_connector" and "sap_suse_cluster_connector""

Btw, at the beginning I only added it to "HA_CLUSTER_INIT: yes" ASCS test case but I noticed [Bug 1229593](https://bugzilla.suse.com/show_bug.cgi?id=1229593) said this tool also support ERS so I added it for both ASCS and ERS. And added the test module to the end of ENSA testing.
for example: https://openqa.suse.de/tests/15263762#

Reply:
Yes, it's supported in both ASCS and ERS, but adding it in both nodes without further changes can lead to the issue I describe above, in particular for the tests related to `smm` and specially for `fra` & `cpa` which will stop resources. I believe it would be possible to add the module in both nodes only if:

1. We introduce some synchronization control via barriers so the stop & start commands are performed first in one node and then in the other in sequence, avoiding the possibility that they run in parallel.
2. Or, if we modify the `smm`, `fra` & `cpa` tests so they only run on one node via `lib/hacluster::is_node()`.

In any case, both options require further changes in the module itself.